packages/cli/src/databases/entities/WebhookEntity.ts

```diff
@@ -18,8 +18,8 @@ export class WebhookEntity implements IWebhookDb {
 	node: string;
 
 	@Column({ nullable: true })
-	webhookId: string;
+	webhookId?: string;
 
 	@Column({ nullable: true })
-	pathLength: number;
+	pathLength?: number;
```

Comment:
Why do these need to be optional now?

Reply:
From what i can tell, these have always been optional. hence the `{ nullable: true }` in the column definition, and nullable columns in the schema. I changed this here to make the entity consistent with the schema, and also to be able to delete the `IWebhookDB` interface.

Reply:
Then the type should be `pathLength?: number | null;`?

Reply:
in typeorm 0.2.x null columns values are returned as `undefined`. When we upgrade to 0.3.x (hopefully soon 🤞🏽) we can then change these to use `|null`.